src/proficiency.h

```diff
@@ -6,6 +6,7 @@
 #include <set>
 #include <vector>
 
+#include <string>
```

Comment:
Please keep the standard library and cataclysm includes separate. Ideally sorted alphabetically too, but that's not necessary.
```suggestion
#include <string>

```